docs/content/en/docs/reference/changelog.md

```diff
@@ -6,6 +6,12 @@ weight: 35
 
 ## Unreleased
 
+## [v0.14.0](https://github.com/aws/eks-anywhere/releases/tag/v0.13.0)
+
+### Added
+- Add support for EKS Anywhere on AWS Snow ([#1090](https://github.com/aws/eks-anywhere/pull/1090))
```

Comment:
Should the mention of AWS Snow support wait until after the release, since it's not coming out until later?

Reply:
from eksa perspective, snow provider is officially added to eks-a code as we removed the feature gate. I think it is fine we mention it in our release log